ArchiSteamFarm/IPC.cs

```diff
@@ -1189,11 +1192,12 @@ private sealed class ASFResponse {
 			[JsonProperty]
 			private readonly Version Version;
 
-			internal ASFResponse(GlobalConfig globalConfig, uint memoryUsage, DateTime processStartTime, Version version) {
-				if ((globalConfig == null) || (memoryUsage == 0) || (processStartTime == DateTime.MinValue) || (version == null)) {
-					throw new ArgumentNullException(nameof(memoryUsage) + " || " + nameof(processStartTime) + " || " + nameof(version));
+			internal ASFResponse(string buildVariant, GlobalConfig globalConfig, uint memoryUsage, DateTime processStartTime, Version version) {
+				if ((string.IsNullOrEmpty(buildVariant) || (globalConfig == null) || (memoryUsage == 0) || (processStartTime == DateTime.MinValue) || (version == null)) {
+					throw new ArgumentNullException(nameof(buildVariant) + " || " + nameof(memoryUsage) + " || " + nameof(processStartTime) + " || " + nameof(version));
```

Comment:
Missing `nameof(globalConfig)`.

Reply:
That one wasn't there in the first place 😂 i'll add it for you 😉

Reply:
Who said that ASF code is flawless? 😉